release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1267,6 +1267,26 @@ sourceStrategy:
+
+* Previously, if a `configmap` that the router deployment depends on is not created, then the router deployment does not progress. With this update, the cluster Operator reports `ingress progressing=true` if the default ingress controller deployment is progressing. This results in users debugging issues with the ingress controller by using the command `oc get co`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2066560[*BZ#2066560*])
+
+* Previously, CoreDNS was based on kubernetes 1.24 while {product-title} 4.12 is based on kubernetes 1.25. With this update, CoreDNS is v1.10.0, new features have been added, and {product-title} 4.12 uses kubernetes 1.25 APIs. (link: https://issues.redhat.com/browse/OCPBUGS-1731[*OCPBUGS-1731*])
+
+* Previously, routers used GO kubernetes libraries that were based on kubernetes 1.24 while {product-title} 4.12 is based on kubernetes 1.25. With this update, GO kubernetes API is v1.25.2, which aligns the router with {product-title} 4.12 that uses kubernetes 1.25 APIs. (link: https://issues.redhat.com/browse/OCPBUGS-1730[*OCPBUGS-1730*])
+
+* Previously, the Cluster Ingress Operator used GO kubernetes libraries that were based on kubernetes 1.24 while {product-title} 4.12 is based on kubernetes 1.25. With this update, GO kubernetes API is v1.25.2, which aligns the Cluster Ingress Operator with {product-title} 4.12 that uses kubernetes 1.25 APIs. (link: https://issues.redhat.com/browse/OCPBUGS-1554[*OCPBUGS-1554*])
+
+* Previously, the Cluster DNS Operator did not reconcile the `openshift-dns` namespace, which is a requirement for {product-title} 4.12 to have pod-security labels. With this update, the Cluster DNS Operator reconciles the `openshift-dns` namespace and the pod security labels are correct ensuring that pods startup with no problems.  (link: https://issues.redhat.com/browse/OCPBUGS-1549[*OCPBUGS-1549*])
+
+*Previously, the `ingresscontroller.spec.tuniningOptions.reloadInterval` did not support decimal points as valid for `reloadInterval`. This caused an `ingress-controller` to become invalid and unable to be deleted. With this update, `ingresscontroller.spec.tuningOptions.reloadInterval` supports decimals and users can delete ingress controllers with previously unsupported time units. (link: https://issues.redhat.com/browse/OCPBUGS-236[*OCPBUGS-236*])
```

Comment:
```suggestion
* Previously, the `ingresscontroller.spec.tuniningOptions.reloadInterval` did not support decimal numerals as valid parameter values because the Ingress Operator internally converts the specified value into milliseconds which was not a supported time unit. This prevented an Ingress Controller from being deleted. With this update, `ingresscontroller.spec.tuningOptions.reloadInterval` now supports decimal numerals and users can delete Ingress Controllers with `reloadInterval` parameter values which were previously unsupported. (link: https://issues.redhat.com/browse/OCPBUGS-236[*OCPBUGS-236*])
```